cmake/Config.cmake.in

```diff
@@ -0,0 +1,18 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+if(ARABICA_HAVE_BOOST)
+    find_dependency(Boost)
+endif()
+
+if(ARABICA_XML_BACKEND STREQUAL USE_EXPAT)
```

Comment:
This file is created in a similar to [configure_file](https://cmake.org/cmake/help/latest/command/configure_file.html) way; it should be `@USE_EXPAT@`.

Reply:
You mean `@ARABICA_XML_BACKEND@`, right?

Reply:
I guess it should be both.

Reply:
`USE_EXPAT` is a string which is one of the possible values of `ARABICA_XML_BACKEND`. I have changed it to use `@ARABICA_XML_BACKEND@` instead now but it still seems to fail with the same problem:
https://travis-ci.com/FredrikAppelros/hunter/jobs/194881832#L1147

Reply:
We are generating `*Config.cmake` file for users. Variable `USE_EXPAT` from Arabica package build should be locked in `*Config.cmake`, hence it should be `@USE_EXPAT@`. Run the build and check the content of the `*Config.cmake` file.

Reply:
As per my previous comment; `USE_EXPAT` is not a variable but a string which we compare the variable `ARABICA_XML_BACKEND` against.

Reply:
> As per my previous comment; USE_EXPAT is not a variable but a string

I got it now, what a poor choice of options... Definitely should be put in quotes then:
- https://cmake.org/cmake/help/latest/policy/CMP0054.html

Reply:
I have wrapped everything in quotes but still no dice. The generated `arabicaConfig.cmake` file contains the following:
```
include(CMakeFindDependencyMacro)

if("")
    find_dependency(Boost)
endif()

if("USE_EXPAT" STREQUAL "USE_EXPAT")
    find_dependency(EXPAT)
endif()

if("USE_EXPAT" STREQUAL "USE_LIBXML2")
    find_dependency(libxml2)
endif()
```
Still the build fails with the same error:
> CMake Error at CMakeLists.txt:16 (add_executable):
  Target "boo" links to target "EXPAT::expat" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?

Any ideas what I am missing?

Reply:
Probably `CONFIG`?
- https://github.com/ruslo/hunter/blob/v0.23.158/examples/Expat/CMakeLists.txt#L15